truetype/truetype.go

```
@@ -345,6 +433,20 @@ func (f *Font) Index(x rune) Index {
 	return 0
 }
 
+// Name returns the NameID value of a Font.
+// Returns "" on error or not found.
+func (f *Font) Name(id NameID) string {
+	valueOffset, valueLength, pid, _ := parseSubtables(f.name, "name", 6, 12, 10, 8, []int{6}, []int{int(id)}, true)
```

Don't ignore the error returned.